client/app/screens/InviteList.js

```diff
@@ -51,14 +51,27 @@ function capitalize(string) {
 }
 
 export default class InviteListScreen extends React.Component {
+  config = {};
+  token = '';
+
   static navigationOptions = {
     title: 'Invites',
   };
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     const ds = new ListView.DataSource({
       rowHasChanged: (r1, r2) => r1 !== r2,
     });
+
+    console.log(
+      'InviteList:\n' +
+        JSON.stringify(props) +
+        '\n\n------------------------------LIST'
+    );
+
+    this.config = props.navigation.state.params.config;
```

Comment:
Do these get used for anything in this component? If not I'd clean them up (and the log statement above).